log/format.go

```diff
@@ -33,19 +33,19 @@ var locationTrims = []string{
 // format output.
 func PrintOrigins(print bool) {
 	if print {
-		atomic.StoreUint32(&locationEnabled, 1)
+		locationEnabled.Store(1)
 	} else {
-		atomic.StoreUint32(&locationEnabled, 0)
+		locationEnabled.Store(0)
 	}
 }
 
 // locationEnabled is an atomic flag controlling whether the terminal formatter
 // should append the log locations too when printing entries.
-var locationEnabled uint32
+var locationEnabled atomic.Uint32
```

Comment:
This should be an `atomic.Bool`, really. It's a small enough change that we might just as well fix it in this PR while we're at it, IMO,